budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/pages/user/selectproject/SelectProjectPage.html

```diff
@@ -34,7 +34,7 @@ <h4>Or select an existing one</h4>
         </div>
         <div class="footer">
             <div class="row">
-                <button wicket:id="backlink2" id="logout" type="button" class="btn bg-olive btn-block">Logout</button>
+                <button wicket:id="logoutLink" id="logout" type="button" class="btn bg-olive btn-block">Logout</button>
```

Comment:
The `id="logout"` should be added to the backLink instead of the logoutLink. This id is used in the script `js/selectProject.js`. Currently this causes the user to be logged out when pressing the escape key instead of going back to the page he was previously on.

Reply:
I changed the behavior in `selectProject.js`: When the user first logs in and presses escape on the Select Project Page, the user is logged out. When a user is already logged in, goes to the Select Project Page and presses escape, the user is redirected back to where he was before that.